conda_smithy/configure_feedstock.py

```diff
@@ -1594,6 +1595,53 @@ def make_jinja_env(feedstock_directory):
     return env
 
 
+def get_migrations_in_dir(migrations_root):
+    res = {}
+    for fn in glob.glob(os.path.join(migrations_root, "*.yaml")):
+        with open(fn, "r") as f:
+            contents = f.read()
+            migration_yaml = (
+                yaml.load(contents, Loader=yaml.loader.BaseLoader) or {}
+            )
+            # Use a object as timestamp to not delete it
+            ts = migration_yaml.get("migrator_ts", object())
+            res[ts] = fn
+    return res
+
+
+def set_migration_fns(forge_dir, forge_config):
+    exclusive_config_file = forge_config["exclusive_config_file"]
+    cfp_migrations_dir = os.path.join(
+        os.path.dirname(exclusive_config_file),
+        "share",
+        "conda-forge",
+        "migrations",
+    )
+
+    migrations_root = os.path.join(forge_dir, ".ci_support", "migrations")
+    migrations_in_feedstock = get_migrations_in_dir(migrations_root)
+
+    if not os.path.exists(cfp_migrations_dir):
+        forge_config["migration_fns"] = list(migrations_in_feedstock.values())
+        return
+
+    migrations_in_cfp = get_migrations_in_dir(cfp_migrations_dir)
+
+    result = []
+    for ts, fn in migrations_in_feedstock.items():
+        if type(ts) == object:
+            # This file doesn't have a timestamp. Use it as it is.
+            result.append(fn)
+        elif ts in migrations_in_cfp:
+            # Use the one from cfp.
+            result.append(migrations_in_cfp[ts])
+        else:
+            # Delete this as this migration is over.
+            remove_file(fn)
+    forge_config["migration_fns"] = result
```

Comment:
I'm confused by this part. I guess we are storing the path in the conda-forge.yml in the feedstock now? Is there a reason to do this? The paths will end up with CI specific bits because they depend on the conda env when it is rendered.

Reply:
Ohhh wait. This is never written back out?

Reply:
Nope. It isn't written back out.